principal/apis/replication/replication.proto

```
@@ -123,4 +143,9 @@ service Replication {
     rpc Status(StatusRequest) returns (ReplicationStatus) {
         option (google.api.http).get = "/api/v1/replication/status";
     }
+
+    // ConfirmMissingResources classifies resources absent from a snapshot before cleanup.
+    rpc ConfirmMissingResources(ConfirmMissingResourcesRequest) returns (ConfirmMissingResourcesResponse) {
+        option (google.api.http).post = "/api/v1/replication/confirm-missing-resources";
```

Buf (1.69.0)

[error] cannot find `google.api.http` in this scope

(COMPILE)